pom.xml

```diff
@@ -110,6 +110,7 @@
         <dep.commons.codec.version>1.17.1</dep.commons.codec.version>
         <aws.sdk.version>2.32.9</aws.sdk.version>
         <release.autoPublish>true</release.autoPublish>
+        <dep.logback.version>1.5.19</dep.logback.version>
```

Comment:
Can you try making the change to airbase instead - https://github.com/prestodb/airbase/pull/36/files is what I had done last time